var/spack/repos/builtin/packages/angsd/package.py

```diff
@@ -16,18 +16,30 @@ class Angsd(MakefilePackage):
     homepage = "https://github.com/ANGSD/angsd"
     url      = "https://github.com/ANGSD/angsd/archive/0.919.tar.gz"
 
+    version('0.933', sha256='2f992325dc08fa25ac525d9300ef6bd61808e74c521b4cc72a2ce00d98f402bb')
     version('0.921', sha256='8892d279ce1804f9e17fe2fc65a47e5498e78fc1c1cb84d2ca2527fd5c198772')
     version('0.919', sha256='c2ea718ca5a5427109f4c3415e963dcb4da9afa1b856034e25c59c003d21822a')
 
     depends_on('htslib')
     conflicts('^htslib@1.6:', when='@0.919')
 
+    depends_on('zlib')
+    depends_on('lzma')
+    depends_on('curl')
+
     def setup_run_environment(self, env):
         env.set('R_LIBS', self.prefix.R)
```

Comment:
Are we missing a run-time dependency on R?

Reply:
`angsd` provides some R tools through this path but does not require R to run.

Reply:
Hmm, I wonder if we should add a variant that optionally adds an R dependency. I don't use this package, so I'll leave it up to you to decide what's best.

Reply:
I went ahead and added an `r` variant enabling the runtime dependency.